examples/framework/preferences/minimalEditor/README.md

```diff
@@ -0,0 +1,9 @@
+
+## Minimal Preferences Editor ##
+
+This folder provides an example of a minimal but fully functional preferences editor. It has:
+- only one preference
+- no enactor
+- no localization (Note that this is not best practice!)
+
+This code is the basis for the Preferences Framework tutorial.
```

Comment:
I wonder if this write up should be on the example index.html page as well (or instead). What do you think?